src/PostgREST/DbStructure.hs

```diff
@@ -43,7 +43,14 @@ import Text.InterpolatedString.Perl6 (q)
 import PostgREST.DbStructureTypes
 import PostgREST.Private.Common
 import           PostgREST.PgVersions (PgVersion(..))
-import PostgREST.Types
+
+
+-- | The source table column a view column refers to
+type SourceColumn = (Column, ViewColumn)
+type ViewColumn = Column
```

Comment:
Should those go to `PostgREST.DbStructure.Table`?

Reply:
Also wondered - or with Relation. But this type is only used within DbStructure.hs, nowhere else - It might be something that we can completely refactor out later down the line, so I would leave it within that module for now.

Reply:
Hm. Right now, you have only moved the types into `DbStructure.Table` (and the others), I think, but not split up the `DbStructure` file itself. My assumption was that this would be the case later and right now we're only aligning the types in the right place.

But I see now, that those two types are really just helpers to the `decodeSourceColumns` and `pfkSourceColumns` functions. So it makes sense to put them together with those two for now.